sln/ODataMigration.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.645
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Extensions.OData.Migration", "..\src\Microsoft.Extensions.OData.Migration\Microsoft.Extensions.OData.Migration.csproj", "{4575BD2C-1117-4CA2-8500-24BBB649FAFB}"
```

Comment:
>Microsoft.Extensions.OData.Migration [](start = 53, length = 36)

Microsoft.Extensions.* is resolved by other teams.

So, we have to change it. Please change it as;

Microsoft.OData.Extensions.Migration

Reply:
changed